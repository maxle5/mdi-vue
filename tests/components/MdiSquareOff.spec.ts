
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiSquareOff from "../../src/components/MdiSquareOff.vue";

test("MdiSquareOff snapshot", () => {
  const wrapper = mount(MdiSquareOff, {});
  expect(wrapper.html()).toMatchSnapshot();
});
