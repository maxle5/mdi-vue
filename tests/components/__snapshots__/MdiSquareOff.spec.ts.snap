// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiSquareOff snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M20.84 22.73L19.11 21H3V4.89L1.11 3L2.39 1.73L22.11 21.46L20.84 22.73M21 3H6.2L21 17.8V3Z" style="fill: currentColor;"></path>
</svg>"
`;
